packages/test/test-end-to-end-tests/src/test/gc/gcStats.spec.ts

```diff
@@ -235,6 +268,36 @@ describeCompat("Garbage Collection Stats", "NoCompat", (getTestObjectProvider) =
 			unrefDataStoreStats.totalBlobSize,
 			"dataStore1 and dataStore2's blobs should be in unreferenced blob size",
 		);
+
+		// Deleted stats. Wait for sweep timeout and send an op to update the current reference timestamp. Usually,
+		// GC wouldn't run without ops so this step is not needed but its needed here because we are explicitly
+		// running GC in absence of ops.
+		await delay(sweepTimeoutMs);
+		mainDataStore._root.set("update", "timestamp");
+		await provider.ensureSynchronized();
+
+		mainContainer.close();
+
+		// Run GC. This will generate a GC sweep op with the sweep ready node ids and wait for the op to be processed.
+		await summarizerRuntime.collectGarbage({});
+		await provider.ensureSynchronized();
+
+		expectedGCStats.nodeCount -= 6;
+		expectedGCStats.unrefNodeCount -= 6;
+		expectedGCStats.deletedNodeCount += 6;
+		expectedGCStats.updatedNodeCount = 0;
+		expectedGCStats.dataStoreCount -= 2;
+		expectedGCStats.unrefDataStoreCount -= 2;
+		expectedGCStats.deletedDataStoreCount += 2;
+		expectedGCStats.updatedDataStoreCount = 0;
```

Comment:
I wonder if it makes more sense to include deletions in the "updated" count?  It's fine either way as long as we remember how it works when looking at the data :)

Reply:
The updated count tells whether a node's reference state changed. It's mainly used to validate that data store are not re-summarized unexpectedly [here](https://github.com/microsoft/FluidFramework/blob/e735861a4a03d23c7287fb7bf6ae24351ee55803/packages/runtime/container-runtime/src/summary/summaryGenerator.ts#L349).
If a data store is deleted, it's reference state doesn't change and it won't be re-summarized so there is no need to update it.
Like you said, we could update its meaning so that deleted in included but I don't think it adds anything yet.